client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -145,13 +145,14 @@
         </template>
         <template v-slot:flyout="{ assignee, id, originalPdf, segmentsCount, synchronized }">
           <dp-flyout>
-            <a
+            <button
```

Comment:
shouldnt this be dpButton?

Reply:
I think `DpButton` doesn't support buttons that look like links yet, does it?

Reply:
my bad